src/Command/Compare.php

```diff
@@ -16,8 +18,12 @@ protected function configure(): void
     {
         $this->setName('compare')
             ->setDescription('Runs tests, normalizes the results then analyzes the results')
-            ->addArgument('run', InputArgument::OPTIONAL, 'The name of the test run, if omitted will be generated from date')
-            ->addArgument('file', InputArgument::OPTIONAL, 'Path to a file to use as the source of useragents rather than test suites')
+            ->setDefinition(
```

Comment:
Could you change the other commands in the same way too?

Reply:
Do you mean using `setDefinition` or changing from Argument to Option?

The reason I changed this one was because it wasn't possible to set just the `file` argument since it was expecting `./bin/console compare <run> <file>`. Now the `file` argument can be specified right after the command, and then `run` option uses a parameter prefix: `./bin/console compare <file> --run=<run>`.

I don't think the other commands have that issue, but I can check.

As for the syntax, I just changed that because it was the example used in the Symfony Console documentation: https://symfony.com/doc/current/components/console/console_arguments.html  I think either syntax is acceptable (the `addArgument` and `addOption` syntax is just short-hand).

Reply:
I mean both changes. For me it's easier to understand what commands do, when they are defined in a similar way.